helm/k8s-pub-path-routing-varnish/app-configs/k8s-pub-path-routing-varnish_eks_publish.yaml

```diff
@@ -0,0 +1,10 @@
+# Values used for the deployed application.
+replicaCount: 2
+service:
+  name: path-routing-varnish
+
+eksCluster: true
+
+image:
```

Comment:
The image config comes from the default Values file. Why do you overwrite it with the same?

Reply:
Removed the extra image declaration